packaging/ubuntu-16.04/rules

```diff
@@ -253,6 +253,9 @@ override_dh_install:
 	rm $(CURDIR)/debian/snapd/$(SYSTEMD_UNITS_DESTDIR)/snapd.apparmor.service
 	rm $(CURDIR)/debian/tmp/usr/lib/snapd/snapd-apparmor
 
+	# snap-preseed should not be distributed yet.
```

Comment:
Can you please also change `packaging/snapd.mk`

Reply:
Done.